manifests/devconsole/0.1.0/devconsole_v1alpha1_component_crd.yaml

```diff
@@ -17,23 +16,29 @@ spec:
     openAPIV3Schema:
       properties:
         apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
           type: string
         kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
         metadata:
           type: object
         spec:
           properties:
             buildType:
+              description: Container image use to build (nodejs, golang etc..)
               type: string
-              description: The type of the image that is required to build your component.
-              enum:
-              - nodejs
-            codebase:
+            gitSourceRef:
+              description: GitSourceRef is the source code of your component. Atm
+                only public remote URL are supported.
```

Comment:
"Only public remote URLs are supported."

Can we avoid abbreviations such as "atm"...?

Maybe better to use "At the moment" Or "Currently, only public remote URLs are supported."